src/Datadog.Trace.OpenTracing/HttpHeadersCodec.cs

```diff
@@ -34,16 +40,9 @@ public void Inject(global::OpenTracing.ISpanContext context, object carrier)
 
             IHeadersCollection headers = new TextMapHeadersCollection(map);
 
-            if (context is OpenTracingSpanContext otSpanContext && otSpanContext.Context is SpanContext ddSpanContext)
-            {
-                // this is a Datadog context
-                SpanContextPropagator.Instance.Inject(ddSpanContext, headers);
-            }
-            else
+            if (context is OpenTracingSpanContext otSpanContext && otSpanContext.Context is SpanContext spanContext)
             {
-                // any other OpenTracing.ISpanContext
```

Comment:
Any particular reason to remove this fallback path?

Reply:
Seems like refactoring error. I'll add back references to DD headers but to be more correct I would add OpenTracing propagator logic.